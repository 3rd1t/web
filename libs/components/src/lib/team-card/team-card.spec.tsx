import React from "react"
import { render } from "@testing-library/react"
import renderer from "react-test-renderer"
import TeamCard from "./team-card"

describe(" TeamCard", () => {
  it("should render successfully", () => {
    const { baseElement } = render(
      <TeamCard
        name="Nicolas Webersinke"
        position="Very important"
        description="Does very important stuff"
        image="https://via.placeholder.com/150"
      />,
    )
    expect(baseElement).toBeTruthy()
  })
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <TeamCard
          name="Nicolas Webersinke"
          position="Very important"
          description="Does very important stuff"
          image="https://via.placeholder.com/150"
        />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
