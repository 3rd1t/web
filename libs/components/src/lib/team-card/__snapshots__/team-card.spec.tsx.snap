// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` TeamCard renders correctly 1`] = `
<div
  className="flex flex-col items-center h-full text-center sm:flex-row sm:justify-start sm:text-left"
>
  <img
    alt="Nicolas Webersinke"
    className="flex-shrink-0 object-cover object-center w-40 h-40 rounded-full shadow-xs"
    src="https://via.placeholder.com/150"
  />
  <div
    className="flex flex-col sm:pl-8"
  >
    <h2
      className="text-lg font-medium text-gray-900"
    >
      Nicolas Webersinke
    </h2>
    <h3
      className="text-gray-500"
    >
      Very important
    </h3>
    <p
      className="max-w-xs mt-3 sm:max-w-none"
    >
      Does very important stuff
    </p>
  </div>
</div>
`;
